ts/Core/Chart/Chart.ts

```diff
@@ -3868,7 +3868,10 @@ class Chart {
                 // Show or hide the Reset zoom button
                 if (displayButton && !this.resetZoomButton) {
                     this.showResetZoom();
-                } else if (!displayButton && this.resetZoomButton) {
+                } else if (
+                    !displayButton && this.resetZoomButton &&
+                        trigger !== 'mousewheel'
```

Comment:
I would suggest splitting it into 3 lines
```suggestion
                    !displayButton && 
                    this.resetZoomButton &&
                    trigger !== 'mousewheel'
```